Model deletion contract rules (#8426)
fishtown-analytics/docs.getdbt.com: website/snippets/_deprecation-date-callout.md

:::info

> If your model has an [enforced contract](/docs/mesh/govern/model-contracts), you won't be able to delete the model until after the `deprecation_date` has passed. dbt won't allow deleting the model(s) with enforced contracts until after their `deprecation_date` to protect downstream consumers.

Reviewer: Some suggestions to use present tense + make it more concise
```suggestion
If your model has an [enforced contract](/docs/mesh/govern/model-contracts), you cannot delete the model until after the `deprecation_date` has passed. dbt doesn't allow deleting models with enforced contracts before their `deprecation_date` to protect downstream consumers.
```
